### ISSUE TITLE
Pass id an index to renderImage callback

### ISSUE THREAD

**Issue by hanno-ferrannegre** [terrysahaidak/react-native-gallery-toolkit#17]
Hey there 👋 

I wanted to propose the following:

```js
const renderImage = useCallback(
  ({source: {uri}, id, index, ...rest}) => (
    <CacheImage
      {...rest}
      uri={uri}
      // This
      testID={`image-${index}`}
      // Or 
      testID={`image-${id}`}
    />
  ),
  [],
);
```

It would be useful if we can get `item.id` in the callback and also `index` in case we want to use `index`.

**Comment by terrysahaidak** [comment]
Hey. I can see why would you want this.
I'm not sure I want to add this information directly to the `renderImage`, because it's coming from the ImageTransformer component that shouldn't know anything about index etc.

But I'll figure something out.

You're using Standalone Gallery?

**Comment by terrysahaidak** [comment]
Could you test these changes?

https://github.com/terrysahaidak/react-native-gallery-toolkit/pull/18

**Comment by ferrannp** [comment]
> You're using Standalone Gallery?

Yes.

> Could you test these changes?
> #18

@terrysahaidak works perfectly 👍

**Comment by terrysahaidak** [comment]
I added this only for standalone gallery

**Comment by ferrannp** [comment]
Thank you. Waiting for the release then 🚀 .

**Comment by terrysahaidak** [comment]
This change is included in 1.0.0-alpha.1 :)
